webapi/tct-2dtransforms-css3-tests/2dtransforms/csswg/transform-blank-ref.html

```diff
@@ -1,10 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This reference file is referenced by

- webapi/tct-3dtransforms-css3-tests/3dtransforms/csswg/transform3d-scale-004-manual.html
- webapi/tct-3dtransforms-css3-tests/3dtransforms/csswg/transform3d-preserve3d-012-manual.html

It should be listed by tests*xml files as reftests.